network/network_windows.go

```diff
@@ -347,6 +393,16 @@ func (nm *networkManager) newNetworkImplHnsV2(nwInfo *NetworkInfo, extIf *extern
 		logger.Info("Network with name already exists", zap.String("name", hcnNetwork.Name))
 	}
 
+	// check if ipv6 default gateway route is missing before windows endpoint creation
+	for i := range nwInfo.Subnets {
+		if nwInfo.Subnets[i].Family == platform.AfINET6 {
+			if err = nm.addIPv6DefaultRoute(); err != nil {
+				// should not block network creation but remind user that it's failed to add ipv6 default route to windows node
+				logger.Error("failed to add missing ipv6 default route to windows node active/persistent store", zap.Error(err))
+			}
```

Comment:
need to add a `break` here to make clear that once the route is added we don't continue looping.